real_time/surface_src/java_src/Alice/src/test/java/com/roboclub/robobuggy/main/RobobuggyConfigFileTest.java

```diff
@@ -0,0 +1,206 @@
+package com.roboclub.robobuggy.main;
+
+import com.roboclub.robobuggy.robots.PlayBackRobot;
+import com.roboclub.robobuggy.robots.SimRobot;
```

Comment:
I do not understand this. We need those imports because the ConfigFile file is the one with the giant switch statement that controls the type of robot we make...